### PR TITLE
Fix issue #33

### PR DESCRIPTION
**Pull request by FrozenPigs**
Changes the language to not reccomend clients sending a prefix to fix issue #33.

### CODE REVIEW

**Comment by ProgVal** [comment]
`Clients MAY (NOT RECCOMENDED)` is equivalent to `Clients SHOULD NOT`, but the latter sounds better IMO. Could you change it?

**Comment by DanielOaks** [comment]
I'd reword both lines a bit more too, maybe this text works? (we can probably just commit this change directly if it looks okay)

> Clients SHOULD NOT include a prefix of their nickname on messages they send. They should avoid doing so because it makes their messages more likely to be misinterpreted or rejected by the server.

**Comment by progval** [comment]
Looks like it's already done here: https://github.com/ircdocs/modern-irc/blame/8fb236197429a834999858bce089d6a6118d833b/index.md#L326
